### DISCUSSION TITLE
How to customise the voice artist and speed?

### DISCUSSION THREAD

**Discussion by gmmkmtgk** [pndurette/gTTS#379] [Q&A]
How to customise the voice artist and speed?
Like gender of the person speaking, accent, etc

plus how to customise the speed of the text

**Comment by phpandroiddeveloper** [comment]
# Set the voice artist and speed
voice = gTTS(text, lang='en', slow=True, voice='en-us-x-sfg#male_2-local')

**Reply by phpandroiddeveloper** [reply]
To customize the voice artist, you can use the lang parameter to specify the language of the text and the voice to use. For example:
Copy code
from gtts import gTTS
voice = gTTS(text, lang='en')  # Default English voice
voice = gTTS(text, lang='es')  # Spanish voice

To customize the speed, you can use the slow parameter. Set it to True to slow down the speed, or False to use the default speed. For example:
Copy code
voice = gTTS(text, slow=True)  # Slowed down speed
voice = gTTS(text, slow=False)  # Default speed

Here is an example of the full code:

Copy code
from gtts import gTTS

# Customize the voice artist and speed
text = "Hello, this is a test of the text-to-speech function."
voice = gTTS(text, lang='en', slow=True)

# Save the audio file
voice.save("speech.mp3")
